src/openrct2/ride/ShopItem.cpp

```diff
@@ -146,7 +145,7 @@ money64 ShopItemGetCommonPrice(Ride* forRide, const ShopItem shopItem)
 
 bool ShopItemHasCommonPrice(const ShopItem shopItem)
 {
-    return (gSamePriceThroughoutPark & EnumToFlag(shopItem)) != 0;
+    return (OpenRCT2::GetGameState().SamePriceThroughoutPark & EnumToFlag(shopItem)) != 0;
```

Comment:
Could you `using namespace OpenRCT2;` under the includes? (Also applies to other cpp files that happen to use something from that namespace.)